deploy/vagrant/provision/playbook.yml

```diff
@@ -3,7 +3,7 @@
   tasks:
     - include: roles/common/tasks/main.yml
     - include: roles/lib/tasks/mount.yml
-      when: provider == "aws"
+      when: (provider == "aws") or (provider == "google")
```

Comment:
I didn't look at the mount task at all. What I've submitted is up and running and passing tests. Now, I'm curious. I'll take a look at the mount task.